vizro-core/changelog.d/20230904_155944_huong_li_nguyen_add_card_styling_back.md

```diff
@@ -0,0 +1,42 @@
+<!--
```

Comment:
Is this file should be in the repo?

Reply:
Yes, this file will be added to all PRs now. In fact, we shouldn't be able to merge without it. I didn't add any changes to the file though, so it will not be reflected in the final CHANGELOG.md.